en/releases/core_components.md

```diff
@@ -0,0 +1,55 @@
+# PX4 Core Components
+
+These core components are *guaranteed* to work with a stable point release of PX4. They form the foundation of the PX4-Autopilot and it's core functionality.
+
+## Core component criteria
+
+Each core component must have:
+- Maintainer
+- Tester
+
+## PX4 Core Components List
+
+| Core component (category) | Sub component | Maintainer | Tester |
+|---|---|---|---|
+| uORB (Architecture) |  |  |  |
+| Parameters (Architecture) |  |  |  |
+| Realtime Clock (Architecture) |  |  |  |
+| Work queue (Architecture) |  |  |  |
+| GNSS (State Estimation) | Velocity / Position |  |  |
+|  | Heading |  |  |
+| Optical flow (State Estimation) |  |  |  |
+| External vision (State Estimation) |  |  |  |
+| Boards (NuttX) | Pixhawk standard |  |  |
```

Comment:
@davids5 How would we categorize the core set of boards we actually can test / support? I would think only the pixhawk standard ones would make sense: https://docs.px4.io/main/en/flight_controller/autopilot_pixhawk_standard.html

What do you think? Also, anything else you would like to add other than boards, in terms of NuttX features?

Reply:
@junwoo091400 I think if it is built regularly and on the test rack and can be supported. so not just the px4 directory .

Reply:
@junwoo091400 I think this is the conversation we are having on defining tiers of supported hardware, and since that conversation is not settled, we should revisit this section later